target/linux/kirkwood/image/Makefile

```diff
@@ -126,6 +126,19 @@ define Device/raidsonic_ib-nas62x0
 endef
 TARGET_DEVICES += raidsonic_ib-nas62x0
 
+define Device/seagate_blackarmor-nas220
+  DEVICE_VENDOR := Seagate
+  DEVICE_MODEL := Blackarmor NAS220
+  DEVICE_DTS := kirkwood-blackarmor-nas220
```

Comment:
I've just pushed a patch to "calculate" DEVICE_DTS automatically from the device node name, so this line can be dropped.

Reply:
Removed the DEVICE_DTS line. Let me know if you need anything else. Thanks